### ISSUE TITLE
Implement Oscilloscope

### ISSUE THREAD

**Issue by mariobehling** [fossasia/pslab-iOS#3]
Please implement the Oscilloscope similar to the Android app.

**Comment by duplixx** [comment]
Hi @mariobehling is there any requirement for developing an iOS app
I would like to work on developing the solution

**Comment by mariobehling** [comment]
@duplixx We discussed that implementing an app with fluttter would be great as it could be used on both mobile systems.

**Comment by duplixx** [comment]
@mariobehling yeah agreed flutter would be better choice, can you share the implementation progress docs or something

**Comment by mariobehling** [comment]
Currently this is in the idea stage.
